src/core/utils.ts

```diff
@@ -62,3 +63,22 @@ export function getWorkspaceName(workspaces: Workspace[], id: string) {
 export function sleep(timeout: number) {
   return new Promise(res => window.setTimeout(res, timeout));
 }
+
+/**
+ * Replace variables in an array of values.
+ * Useful for multi-value variables.
+ */
+export function replaceVariables(values: string[], templateSrv: TemplateSrv) {
+  const replaced: string[] = [];
+  values.forEach((col: string, index) => {
+    let value = col;
+    if (templateSrv.containsTemplate(col)) {
+      const variables = templateSrv.getVariables() as any[];
```

Comment:
Why do we need the `any[]` cast here?

Reply:
`current.value` isn't publicly accessible on the TypedVariableModel class. We do the same on line 51 of this file as well.